website/docs/r/email_communication_service_domain_sender_username.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Communication"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_email_communication_service_domain_sender_username"
+description: |-
+  Manages an Email Communication Service Domain Sender Username.
+---
+
+# azurerm_email_communication_service_domain_sender_username
+
+Manages an Email Communication Service Domain Sender Username.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_email_communication_service" "example" {
+  name                = "example-emailcommunicationservice"
+  resource_group_name = azurerm_resource_group.example.name
+  data_location       = "United States"
+}
+
+resource "azurerm_email_communication_service_domain" "example" {
+  name              = "AzureManagedDomain"
```

Comment:
could the value of `name` start with `example-` ?

Reply:
@ms-zhenhua , Nope. Service API requires that the name must be AzureManagedDomain.